asdf/tags/time/tests/test_time.py

```diff
@@ -103,6 +102,15 @@ def test_time(tmpdir):
     helpers.assert_roundtrip_tree(tree, tmpdir)
 
 
+def test_time_with_location(tmpdir):
+    # See https://github.com/spacetelescope/asdf/issues/341
+    t = time.Time([1,2], location=[[1,2], [3,4], [5,6]], format='cxcsec')
```

Comment:
If the schema defines the `x`, `y`, `z` of `location` to be quantities, then why does this pass?

Reply:
Good question: it's because the list that gets passed to the constructor of `time.Time` gets converted into an `EarthLocation` object, which uses `Quantity` for each of the coordinate components. This test could have also explicitly created an `EarthLocation` object and passed it to the `Time` constructor, but since the conversion is done automatically there's no need.

Reply:
I see. The example is a bit confusing because it's unclear that default units in coordinates are used. But it works.

Reply:
Good point. I updated it so hopefully it's more clear now.